src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Confirm.tsx

```diff
@@ -264,13 +285,29 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
     })()
   }, [accountId, assetId, chainId, fromAddress])
 
+  const memo = useMemo(() => {
+    if (isRunePool && state) {
+      const balanceCryptoPrecision = bnOrZero(amountAvailableCryptoPrecision.toPrecision())
+      const percent = bnOrZero(state.withdraw.cryptoAmount)
+        .div(balanceCryptoPrecision)
+        .times(100)
+        .toFixed(0)
+      const basisPoints = convertPercentageToBasisPoints(percent)
+
+      return `pool-:${basisPoints}`
+    }
+    if (quote?.memo) return quote.memo
```

Comment:
stylistic: Prefer early returning this guy to keep the meaty logic under

Reply:
Fixed in [1f9d825](https://github.com/shapeshift/web/pull/7449/commits/1f9d8254294f6843bde971347039e344909ac032)